charts/partials/templates/_proxy-init.tpl

```diff
@@ -41,7 +41,7 @@ imagePullPolicy: {{.Values.proxyInit.image.pullPolicy | default .Values.imagePul
 name: linkerd-init
 {{ include "partials.resources" .Values.proxyInit.resources }}
 securityContext:
-  {{- if or .Values.proxyInit.closeWaitTimeoutSecs .Values.proxyInit.runAsRoot }}
+  {{- if or .Values.proxyInit.closeWaitTimeoutSecs .Values.proxyInit.privileged }}
```

Comment:
I'm curious: what is the relationship between `.Values.proxyInit.closeWaitTimeoutSecs` and this feature?

Reply:
`closeWaitTimeoutSecs` is there to address long-running requests (predates me, #4276 should have info about it). The feature can't be used without running in privileged mode, which is why it acts as a conditional in our templates. When the change to run the init container as non-root was made, we've chosen to tackle the issue through templates (https://github.com/linkerd/linkerd2-proxy-init/pull/49#issuecomment-954635072) and this is since tied to anything that has to do with init container privileges.